src/app/Core/ExportServices/generateParameterHeader.service.ts

```diff
@@ -20,10 +20,10 @@ export function generateParameterHeader(
     HeaderRows.push("\uFEFF");
     HeaderRows.push("#Parameter:");
 
-    mainFilterAttributes.forEach((element): void => {
+    mainFilterAttributes.forEach((element: string): void => {

```

Comment:
Hint: As stated in some reviews before: forEach(element => {...}) would be perfectly fine. The correct type will automatically be inferred.

Reply:
Done.